sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -111,17 +120,22 @@ AZ_NODISCARD az_result az_iot_hub_client_init(
  * @brief Gets the MQTT user name.
  *
  * The user name will be of the following format:
- * [Format without module id] {iothubhostname}/{device_id}/?api-version=2018-06-30&{user_agent}
- * [Format with module id]
- * {iothubhostname}/{device_id}/{module_id}/?api-version=2018-06-30&{user_agent}
+ *
+ * **Format without module id**

```

Comment:
Here/elsewhere in the repo.
```suggestion
 * **Format without module ID**
```